sdk/cosmosdb/cosmos/test/internal/session.spec.ts

```diff
@@ -120,9 +124,9 @@ describe.skip("Session Token", function (this: Suite) {
     const container2 = clientB.database(dbId).container(containerId);
     await Promise.all([createItem(container), createItem(container2)]);
 
-    // Create an item using client
-    const id = await createItem(container);
-    const { resource, statusCode } = await container2.item(id).read();
+    // Container read with consistency level and max integrated cache staleness(milliseconds)
+    const container3 = clientC.database(dbId).container(containerId);
+    const { resource, statusCode } = await container3.read({ maxIntegratedCacheStaleness: 2000 });
```

Comment:
Nice, how is it verified that the integrated cache header is sent over wire?

One possible flow is to hook at the request-pipeline level and assert that the real header is going through.

Reply:
Using sinon.spy to intercepts the request and check if maxIntegratedCacheStaleness was inside the header.